### PR TITLE
Merge release/1.x into develop

### PR DESCRIPTION
**Pull request by adidahiya**
DO NOT SQUASH

### CODE REVIEW

File: packages/docs-app/src/components/navbarActions.tsx [hunk 1]
```diff
@@ -63,9 +63,20 @@ export class NavbarActions extends React.PureComponent<INavbarActionsProps, {}>
...
             <MenuItem href={version.url} key={index} label={version.version} target="_blank" text={version.name} />
-        ));
+        );
+        const COMPONENT_PACKAGES = [
+            "@blueprintjs/core",
+            "@blueprintjs/datetime",
+            "@blueprintjs/table",
+            "@blueprintjs/labs",
+        ];
+        const libs = releases.filter(({ name }: IPackageInfo) => COMPONENT_PACKAGES.indexOf(name) >= 0).map(renderItem);
```
Comment (giladgray) [inline packages/docs-app/src/components/navbarActions.tsx]: shouldn't need `: PackageInfo` param type

File: packages/docs-app/src/index.tsx [hunk 2]
```diff
@@ -20,10 +20,12 @@ const releases = releasesData.map((pkg: IPackageInfo) => {
     return pkg;
 });
 
-const versions = versionsData.map((version: string) => ({
-    url: `https://palantir.github.io/blueprint/docs/${version}`,
-    version,
-}));
+const versions = Object.keys(versionsData).map((majorVersion: string) => {
```
Comment (giladgray) [inline packages/docs-app/src/index.tsx]: shouldn't need `: string`

File: packages/docs-app/src/components/navbarActions.tsx (hunk 1 above)
Comment (adidahiya) [reply packages/docs-app/src/components/navbarActions.tsx]: pretty sure you almost always do when destructuring in lambdas?

Comment (adidahiya) [reply packages/docs-app/src/components/navbarActions.tsx]: nvm you're right

**Pull request merged**